### PR TITLE
Fix multiple typos and adjust some sentences

### LINKED ISSUE #222: Typo on page 33
Right at the beginning of Module 3, on the question starting with “Can the vectors…“, only one vector is introduced, so it should be singular.

### PR DESCRIPTION
**Pull request by hassanelsheikha**
Hi @siefkenj! I wanted to adjust some sentences and fix some typos in the book. The (simple) changes can be seen in the diff. I would be happy to go back and adjust anything you would like changed. Thank you!

This PR also addresses and closes #222

### CODE REVIEW

File: book/modules/module6.tex [hunk 1]
```diff
@@ -115,7 +115,7 @@
 
 	$\mathcal W$ is \emph{not} a subspace. To see this, notice that $\vec v=\mat{1\\1}\in \mathcal W$,
 	but $0\vec v=\vec 0\notin\mathcal W$. Therefore, $\mathcal W$ is not closed under scalar multiplication
-	and so cannot be a subspace.
+	and so it cannot be a subspace.
```
Comment (siefkenj) [inline book/modules/module6.tex:118]: I don't think `it` is needed.

File: book/modules/module10-exercises.tex [hunk 2]
```diff
@@ -101,7 +101,7 @@
 		\end{solution}
 
 		\prob Let $\mathcal A: \R^{2}\to \R^{2}$ and
-		$\mathcal B: \R^{2}\to \R^{2}$ be matrix transformations with matrices
+		$\mathcal B: \R^{2}\to \R^{2}$ be matrix transformations with respective matrices
```
Comment (siefkenj) [inline book/modules/module10-exercises.tex:104]: I don't think `respective` clarifies anything here.

**Comment by siefkenj** [comment]
Thanks for the fixes! I don't think adding `respective` is helpful for the readers. I've added a couple other comments as well.

File: book/modules/module6.tex (hunk 1 above)
Comment (hassanelsheikha) [reply book/modules/module6.tex:118]: I think putting a subject pronoun before the verb makes the subject clear. The sentence is less confusing this way. If you strongly prefer the original I can revert.

**Comment by hassanelsheikha** [comment]
I've removed all instances of "respective" as requested. Please kindly have a look at my response to the other issue you raised. Thanks for your feedback!

**Comment by siefkenj** [comment]
Thanks!

**Pull request merged**